sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Release History
 ===============
+12.1.0
```

Comment:
Nit - add a blank line above the version like the previous entries (and everywhere else)